CMakeLists.txt

```diff
@@ -219,6 +219,12 @@ add_test(NAME cpp_clang_format COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/clang_format
 # Add test which checks all files have a valid license
 add_test(NAME reuse_lint COMMAND reuse lint)
 
+# Add test which checks that a build rerun doesn't compile and uses caches
```

Comment:
Maybe one last very minor nitpick, if I may: "doesn't compile" sounds to me like literally not compiling, as in the build fails with a compilation error, which is obviously not what is meant here. Maybe something like this is clearer:
```diff
-# Add test which checks that a build rerun doesn't compile and uses caches
+# Add test which checks that a build rerun doesn't recompile and uses caches instead
```

Reply:
Done!